README.md

```diff
@@ -1 +1,33 @@
 # Bubble-Sort
+
+## Project Description
+
+This project involves creating methods to sort a group of items in an array by using the bubble sort techniques. A method was created to sort integers and another one to sort strings based on their lengths.
```

Comment:
Be specific, another bubble_sort_by, that is based on the given block, not length.